js/cdax.js

```diff
@@ -30,7 +30,6 @@ module.exports = class cdax extends Exchange {
                 'createOrder': true,
                 'fetchBalance': true,
                 'fetchClosedOrders': true,
-                'fetchCurrencies': false,
```

Comment:
This edit breaks CDAX:

- https://app.travis-ci.com/github/ccxt/ccxt/builds/243226878#L2904

Reply:
We most likely have to remove or rewrite fetchCurrencies in CDAX.

Reply:
The reason for this is that CDAX and Huobi JP were both inherited from Huobi initially, and then split into separate exchanges.